source/_integrations/speedtestdotnet.markdown

```diff
@@ -64,6 +64,26 @@ automation:
 
 {% endraw %}
 
+### Using as a button to manually run a speed test
+
+{% raw %}
+
+```yaml
+
+type: button
+name: Test Speed
+tap_action:
+  action: call-service
+  service: homeassistant.update_entity
+  target:
+    entity_id:
+      - sensor.speedtest_download
+      - sensor.speedtest_upload
+      - sensor.speedtest_ping
```

Comment:
This will update the speed test 3 times... I don't think that was your intention?